client/src/pages/home/views.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { IIssueAggregate, IPitch, IUser } from 'ssw-common';
+
+import {
+  FieldTag,
+  PitchInterests,
+  PitchTeams,
+  IssueInterests,
+} from '../../components';
+import { ColumnType } from '../../components/Tables/DynamicTable/types';
+import { buildColumn } from '../../components/Tables/DynamicTable/util';
+
+import { Tab, TABS } from './helpers';
+
+const getColumnsForTab = (user: IUser, tab: Tab): ColumnType<any>[] => {
+  switch (tab) {
+    case TABS.MEMBER_PITCHES:
+      return getMemberPitchesView(user);
+    case TABS.SUBMITTED_CLAIMS:
+      return getSubmittedClaimsView(user);
+    case TABS.SUBMITTED_PITCHES:
+      return getSubmittedPitchesView();
+    case TABS.SUBMITTED_PUBLICATIONS:
+      return getSubmittedPublicationsView();
+    default:
+      return [];
+  }
+};
+
+const stringArraySorter = <T extends Array<any>>(a1: T, a2: T): number =>
+  a1.length - a2.length ||
+  a1.reduce((sum, e, i) => sum + e.localeCompare(a2[i]), 0);
+
+const getMemberPitchesView = (user: IUser): ColumnType<IPitch>[] => [
+  titleColumn,
+  associatedTopicsColumn,
+  {
+    title: "Team(s) You're On",
+    width: '1',
+    sorter: (p1, p2) =>
+      stringArraySorter(
+        p1.assignmentContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.teams ?? [],
+        p2.assignmentContributors.find(
+          (contributor) => contributor.userId === user._id,
+        )?.teams ?? [],
+      ),
+    extractor: function TeamsCell(pitch) {
+      return <PitchTeams pitch={pitch} user={user} assignmentContributors />;
+    },
+  },
+  {
+    title: 'Deadline',
+    width: '1',
+    sorter: (p1, p2) =>
+      new Date(p1.deadline).getTime() - new Date(p2.deadline).getTime(),
+    extractor: (pitch) => new Date(pitch.deadline).toLocaleDateString(),
+  },
+];
+
+const getSubmittedPitchesView = (): ColumnType<IPitch>[] => [
+  titleColumn,
+  descriptionColumn,
+  {
+    ...associatedTopicsColumn,
+    width: '2',
+  },
+  {
+    title: 'Status',
+    width: '1',
+    sorter: (p1, p2) => p1.status.localeCompare(p2.status),
+    extractor: function StatusCell({ status }) {
+      return <FieldTag content={status} />;
```

Comment:
It appears that this status tag is a different font size than other parts of the table

Reply:
Also a status of `Approved` isn't giving me a color.